containers/Dockerfile.base-spack

```diff
@@ -60,6 +60,8 @@ ARG SPACK_ARCH=linux-rocky8-x86_64
 ARG COMPILER_PACKAGE=intel-oneapi-compilers
 ARG COMPILER_VERSION=2021.2.0
 ARG COMPILER_NAME=intel
+ARG MPI_NAME=openmpi
+ARG MPI_VERSION=4.0.2
```

Comment:
If these `ARGS` are only used in the `dev` image, perhaps they should be defined there instead - after the `FROM base-spack as dev` part, maybe?